docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -79,96 +77,84 @@ chown root:git /etc/gitea
 
 ### Configure Gitea's working directory
 
-**NOTE:** If you plan on running Gitea as a Linux service, you can skip this step as the service file allows you to set `WorkingDirectory`. Otherwise, consider setting this environment variable (semi-)permanently so that Gitea consistently uses the correct working directory.
-```
+**NOTE:** If you plan on running Gitea as a Linux service, you can skip this step, as the service file allows you to set `WorkingDirectory`. Otherwise, consider setting this environment variable (semi-)permanently so that Gitea consistently uses the correct working directory.
+```sh
 export GITEA_WORK_DIR=/var/lib/gitea/
 ```
 
-### Copy Gitea binary to global location
-
-```
+### Copy the Gitea binary to a global location
+```sh
 cp gitea /usr/local/bin/gitea
 ```
 
 ## Running Gitea
-
-After the above steps, two options to run Gitea are:
+After you complete the above steps, you can run Gitea two ways:
 
 ### 1. Creating a service file to start Gitea automatically (recommended)
-
 See how to create [Linux service]({{< relref "run-as-service-in-ubuntu.en-us.md" >}})
 
 ### 2. Running from command-line/terminal
-
-```
+```sh
 GITEA_WORK_DIR=/var/lib/gitea/ /usr/local/bin/gitea web -c /etc/gitea/app.ini
 ```
 
 ## Updating to a new version
-
 You can update to a new version of Gitea by stopping Gitea, replacing the binary at `/usr/local/bin/gitea` and restarting the instance.
-The binary file name should not be changed during the update to avoid problems
-in existing repositories.
+The binary file name should not be changed during the update to avoid problems in existing repositories.
 
-It is recommended you do a [backup]({{< relref "doc/usage/backup-and-restore.en-us.md" >}}) before updating your installation.
+It is recommended that you make a [backup]({{< relref "doc/usage/backup-and-restore.en-us.md" >}}) before updating your installation.
 
 If you have carried out the installation steps as described above, the binary should
 have the generic name `gitea`. Do not change this, i.e. to include the version number.
 
 ### 1. Restarting Gitea with systemd (recommended)
-
-As explained before, we recommend to use systemd as service manager. In this case ```systemctl restart gitea``` should be enough.
+As we explained before, we recommend to use systemd as the service manager. In this case, `systemctl restart gitea` should be fine.
 
 ### 2. Restarting Gitea without systemd
+To restart your Gitea instance, we recommend to use SIGHUP signal. If you know your Gitea PID, use `kill -1 $GITEA_PID`, otherwise you can use `killall -1 gitea`.
```

Comment:
```suggestion

To restart your Gitea instance, we recommend to use SIGHUP signal. If you know your Gitea PID, use `kill -1 $GITEA_PID`, otherwise you can use `killall -1 gitea`.
```